routes/test_route.py

```
@@ -73,3 +77,52 @@ def create_schedule_for_test(request: ScheduleCreateRequest, db: Session = Depen
+    except Exception as e:
+        return JSONResponse(
+            content={
+                "message": "redis connection failed",
+                "error": str(e)
+            },
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR
+        )
+
+
+@test_router.get("/stt-token")
+def get_stt_token(redis: Redis=Depends(get_redis_client)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)